pkgs/tools/compression/zfp/default.nix

```diff
@@ -0,0 +1,64 @@
+{ cmake, cudatoolkit, fixDarwinDylibNames, fetchFromGitHub, gfortran, lib, llvmPackages, pythonPackages, stdenv, targetPlatform
+, enableCfp ? true
+, enableCuda ? builtins.elem targetPlatform.system cudatoolkit.meta.platforms
+, enableExamples ? true
+, enableFortran ? builtins.elem targetPlatform.system gfortran.meta.platforms
+, enableOpenMp ? true
+, enablePython ? true
+, enableUtilities ? true }:
+
+stdenv.mkDerivation rec {
+  pname = "zfp";
+  version = "0.5.5";
+
+  src = fetchFromGitHub {
+    owner = "LLNL";
+    repo = "zfp";
+    rev = version;
+    sha256 = "19ycflz35qsrzfcvxdyy0mgbykfghfi9y5v684jb4awjp7nf562c";
+  };
+
+  nativeBuildInputs = [ cmake ]
+    ++ lib.optional targetPlatform.isDarwin fixDarwinDylibNames
+    ++ lib.optional enableFortran gfortran
+    ++ lib.optional enableOpenMp llvmPackages.openmp
+    ++ lib.optional enablePython pythonPackages.python;
+
+  buildInputs = lib.optional enableCuda cudatoolkit
+    ++ lib.optional enablePython [ pythonPackages.cython pythonPackages.numpy ];
+
+  cmakeFlags = [
+    # More tests not enabled by default
+    ''-DZFP_BUILD_TESTING_LARGE=ON''
+  ]
+    ++ lib.optional enableCfp "-DBUILD_CFP=ON"
+    ++ lib.optional enableCuda "-DZFP_WITH_CUDA=ON"
+    ++ lib.optional enableExamples "-DBUILD_EXAMPLES=ON"
+    ++ lib.optional enableFortran "-DBUILD_ZFORP=ON"
+    ++ lib.optional enableOpenMp "-DZFP_WITH_OPENMP=ON"
+    ++ lib.optional enablePython "-DBUILD_ZFPY=ON"
+    ++ (if enableUtilities then [ "-DBUILD_UTILITIES=ON" ] else [ "-DBUILD_UTILITIES=OFF" ]);
+
+  # makeFlags = lib.optional stdenv.isDarwin "LDFLAGS=-Wl,-install_name,$(out)/lib/libzfp.${version}.dylib";
```

Comment:
What is with this? Can this be removed? Did you forget to fix something?

Reply:
I think that’s just left over from when I didn’t know how to properly fixLibraryNames for Darwin. It can be removed.